openrtb_ext/request_wrapper.go

```diff
@@ -44,8 +44,15 @@ type RequestWrapper struct {
 
 const (
 	jsonEmptyObjectLength               = 2
-	ConsentedProvidersSettingsStringKey = "ConsentedProvidersSettings"
-	ConsentedProvidersSettingsListKey   = "consented_providers_settings"
+	consentedProvidersSettingsStringKey = "ConsentedProvidersSettings"
+	consentedProvidersSettingsListKey   = "consented_providers_settings"
+	consent                             = "consent"
```

Comment:
@guscarreon I like the change from hardcoded strings to constants. Thanks for doing that. However, one minor comment, for these constant names we should decide on whether we want the `Key` suffix for them or not and keep it consistent among all constant names that represent a hardcoded string. I personally lean towards having the suffix but don't feel too strongly about it so will leave it up to you.